reflection_refraction.py

```diff
@@ -0,0 +1,136 @@
+from distutils.util import strtobool
```

Comment:
Mr. Shopov want's me to add a GUI interface with boxes and buttons. Can you recommend me a library for those things?

Reply:
I don't have lots of experience with python GUIs. [This article ](https://towardsdatascience.com/top-10-python-gui-frameworks-for-developers-adca32fbe6fc) might help you further.

Reply:
[This](http://physics.bu.edu/~duffy/semester2/c24_refraction.html) might inspire you for a GUI.

Reply:
Thank you!

Reply:
[This](https://physics.bu.edu/~duffy/semester2/c24_refraction.html) is exactly what I want